packages/contracts/package.json

```diff
@@ -7,51 +7,46 @@
     "dist/types/*.ts",
     "artifacts/contracts/**/*.json",
     "deployments/**/*.json",
-    "OVM",
-    "iOVM",
-    "libraries",
-    "mockOVM"
+    "chugsplash",
```

Comment:
We forgot to update these as part of the previous cleanup PR that was just merged.